recipes/python/cython.yaml

```diff
@@ -0,0 +1,20 @@
+inherit: ["python3-pkg", "strip"]
+
+metaEnvironment:
+    PKG_VERSION: "0.29.37"
+    PKG_LICENSE: "Apache-2.0"
+
+checkoutSCM:
+    scm: url
+    url: "https://files.pythonhosted.org/packages/2a/97/\
```

Comment:
Hmm...is this the same `cython` as https://github.com/cython/cython/ where the latest release is 3.0.12?

Reply:
Yes. I think I read somewhere that it needs at least Python 3.12 or 3.13 but cannot find that reference any more. :shrug: Let's see what happens when updating to the latest version...

Reply:
:+1: I'm just asking because I'm preparing `lxml`-recipe where `cython` is also needed. I'm using 3.0.12 and it works fine.

Reply:
I've bumped the version to 3.0.12. Seems to compile fine. Let me know if I can merge it...

Reply:
lxml also works :ship: